### ISSUE TITLE
[bug]: Sats remain pending

### ISSUE THREAD

**Issue by MichaelAntonFischer** [lightningnetwork/lnd#8001]
### Background

Several UTXOs stuck.

### Your environment

* version of `lnd` 0.16.4
* which operating system (`uname -a` on *Nix) Debian 11
* version of `btcd`, `bitcoind`, or other backend bitcoind/neutrino
* any other relevant environment details

### Steps to reproduce

I have previously had the issue that on my neutrino LND node, some sats would be stuck after channel closes. Normally briefly switching back to bitcoind backend would resolve the issue, but now I have quite a lot of sats stuck for weeks even though I have been back to bitcoind for 5 days since.

{
	"utxos": [
		{
			"address_type": 4,
			"address": "bc1px3c7hh9j74t48fpzpn00ajtfrcp0t7aj3jguqdu39wp6a0203husfg5te9",
			"amount_sat": 40160,
			"pk_script": "51203471ebdcb2f55753a4220cdefec9691e02f5fbb28c91c037912b83aebd4f8df9",
			"outpoint": "4ea9f65089e050ac575c0291ea069685c543b793f6bee9d64533b41e839bed13:0",
			"confirmations": 0
		},
		{
			"address_type": 4,
			"address": "bc1p9n5spsv7wj4l9ddmur86l3uxrvsfu9a3vl8x8mcdth46rxc6tk5sklcs75",
			"amount_sat": 5007553,
			"pk_script": "51202ce900c19e74abf2b5bbe0cfafc7861b209e17b167ce63ef0d5deba19b1a5da9",
			"outpoint": "f137ecf97a0a7dbfce7dad6f3b7c7cde78586a7869d40e7684347adf6de6eb9f:0",
			"confirmations": 0
		},
		{
			"address_type": 4,
			"address": "bc1pfgj6uwlz5enhkynvk4vuvvp2q3mgqujs90hhzw40qsm7vl6l8wvqdag6ej",
			"amount_sat": 264601,
			"pk_script": "51204a25ae3be2a6677b126cb559c6302a04768072502bef713aaf0437e67f5f3b98",
			"outpoint": "a7dfabc956af0b11fd693a05cf62edba4a42838a269c5b00ab666fc5240fa4b8:0",
			"confirmations": 0
		},
		{
			"address_type": 4,
			"address": "bc1p9tymg3l5zqp0rk6702u0eg05ypk5e5pzzrwswdt9zym7kruwd0wshsex8a",
			"amount_sat": 146312,
			"pk_script": "51202ac9b447f41002f1db5e7ab8fca1f4206d4cd02210dd0735651137eb0f8e6bdd",
			"outpoint": "2dc11501735bc56dcfd57feb278d92801dc180cc167975dbf0a6244f296cb5ff:0",
			"confirmations": 0
		},

### Expected behaviour

Tell us what should happen

### Actual behaviour

Tell us what happens instead

**Comment by bitromortac** [comment]
Do those UTXOs belong to any sweeps `lncli wallet listsweeps --verbose` or can you link them to your pending closes?

**Comment by MichaelAntonFischer** [comment]
Yes, but to be more specific the sats are shown as pending the channels are shown as closed.
				{
					"tx_hash": "f137ecf97a0a7dbfce7dad6f3b7c7cde78586a7869d40e7684347adf6de6eb9f",
					"amount": 147,
					"time_stamp": 1693944440,
					"dest_addresses": [
						"bc1p9n5spsv7wj4l9ddmur86l3uxrvsfu9a3vl8x8mcdth46rxc6tk5sklcs75"
					],
					"output_details": [
						{
							"output_type": 9,
							"address": "bc1p9n5spsv7wj4l9ddmur86l3uxrvsfu9a3vl8x8mcdth46rxc6tk5sklcs75",
							"pk_script": "51202ce900c19e74abf2b5bbe0cfafc7861b209e17b167ce63ef0d5deba19b1a5da9",
							"amount": 5007553,
							"is_our_address": true
						}
					],
					"raw_tx_hex": "02000000000102f4e3b1c45e3177c8230adf9514799f735293766142945637f2d67bdb87fc658b00000000000000000077569708e2cde2407ddd4bbcd816fa99a09b73b40e844ee3eec18c5eebbce01b00000000000000000001c1684c00000000002251202ce900c19e74abf2b5bbe0cfafc7861b209e17b167ce63ef0d5deba19b1a5da902473044022005f6e5935faf297973717a89d818cac8a34bc371ef9caaf13653d2f0f184fbae0220409997bdc926f1ceabfa277c481ce58b244e463fbea7c71910c7fd32c9836a6501282103d3beb288392cf2c033d3b7c5e48911db4208e139a8d71679f81c2f95b430a171ac736460b2680140651cbb4fc083dc479e92a18aca36f90f913627754b2296e94d685fa09cae3f34b9526169ea3de9fdee1dee3ebc593cd5fefc68a0f246453148c2df94c8b56bd1dd4d0c00",
					"label": "0:sweep",
					"previous_outpoints": [
						{
							"outpoint": "8b65fc87db7bd6f23756944261769352739f791495df0a23c877315ec4b1e3f4:0"
						},
						{
							"outpoint": "1be0bceb5e8cc1eee34e840eb4739ba099fa16d8bc4bdd7d40e2cde208975677:0",
							"is_our_output": true
						}
					]
				},

**Comment by bitromortac** [comment]
The sweep you posted is another instance of a foreign anchor sweep interfering with the internal sweep (see this outpoint https://mempool.space/tx/8b65fc87db7bd6f23756944261769352739f791495df0a23c877315ec4b1e3f4#vout=0). The external sweep is still unconfirmed (https://mempool.space/tx/443bdac052fc92b94137c3cb0dbb140cded7acb7cc307348feff84fb42d6f428).

**Comment by MichaelAntonFischer** [comment]
So do I resolve the issue?

**Comment by MichaelAntonFischer** [comment]
Hi,

looks like the transaction was never confirmed and is gone from the mempool. Is there anyway to trigger LND to do the sweep again?

**Comment by ascisco** [comment]
I am having the same issue

> Hi,
> 
> looks like the transaction was never confirmed and is gone from the mempool. Is there anyway to trigger LND to do the sweep again?

**Comment by Opago-Pay** [comment]
I used an online tool to rebroadcast and fee bump the transaction.

**Comment by yyforyongyu** [comment]
This should be fixed by #9609 now, feel free to reopen if it occurs again.
